website/views/shared/profiles/stats/active-pet-and-mount.jade

```diff
@@ -0,0 +1,5 @@
+div(ng-show='user._id != profile._id')
+  h4
+  table.table.table-striped
+    +basicRow('activePet', '{{::profile.items.currentPet ? (isBasicAnimal(profile.items.currentPet, \'pet\') ? env.t(\'petName\', {potion: Content.hatchingPotions[petPotion].text(), egg: Content.eggs[petEgg].text()}) : env.t(Content.specialPets[profile.items.currentPet])) : env.t(\'noActivePet\')}}')
```

Comment:
Hmm weird that it wouldn't return anything.  It seems like this could be done with a few if-statements to check pet-vs-mount and basic-vs-special and return the appropriate strings.